hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -342,6 +342,14 @@
     </description>
   </property>
 
+  <property>
+    <name>dfs.namenode.choosereplicatodelete.considerLoad</name>
+    <value>false</value>
+    <description>Decide if chooseReplicaToDelete considers the target's load or
```

Comment:
Please add a note on how the load is being determined. which metric is being used as the load.